cns/restserver/util.go

```diff
@@ -616,6 +616,37 @@ func (service *HTTPRestService) getNetworkContainerDetails(networkContainerID st
 	return containerDetails, containerExists
 }
 
+func (service *HTTPRestService) getAllNetworkContainersDetails(networkContainersID []string) (map[string]cns.ContainerStatus, bool) {
+	service.RLock()
+	defer service.RUnlock()
+
+	allNcExist := true
+	containerDetails := make(map[string]cns.ContainerStatus)
+
+	for _, ncID := range networkContainersID {
+		ncDetails, ncExists := service.state.ContainerStatus[ncID]
+		if !ncExists {
+			allNcExist = false
+			break
+		}
+
+		containerDetails[ncID] = storeNcStatus(ncDetails)
+	}
+
+	return containerDetails, allNcExist
+}
+
+func storeNcStatus(ncDetails containerstatus) cns.ContainerStatus {
+	var containerDetails cns.ContainerStatus
+	containerDetails.ID = ncDetails.ID
+	containerDetails.VMVersion = ncDetails.VMVersion
+	containerDetails.HostVersion = ncDetails.HostVersion
+	containerDetails.CreateNetworkContainerRequest = ncDetails.CreateNetworkContainerRequest
+	containerDetails.VfpUpdateComplete = ncDetails.VfpUpdateComplete
+
+	return containerDetails
```

Comment:
pls inline and use the short form for struct instantiation:
```suggestion
	return cns.ContainerStatus{
		ID: ncDetails.ID,
		VMVersion: ncDetails.VMVersion,
		HostVersion: ncDetails.HostVersion,
		CreateNetworkContainerRequest: ncDetails.CreateNetworkContainerRequest,
		VfpUpdateComplete: ncDetails.VfpUpdateComplete,
	}
```

Reply:
Fixed.